pkg/cmd/server/api/v1/conversions.go

```diff
@@ -29,6 +29,7 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 				obj.PolicyConfig.OpenShiftInfrastructureNamespace = bootstrappolicy.DefaultOpenShiftInfraNamespace
 			}
 			if len(obj.RoutingConfig.Subdomain) == 0 {
+				// TODO: use dnsDomain instead of cluster.local
```

Comment:
Should also add defaulting for the new config field when it is empty to be set to cluster.local